tests/10apidoc/33room-members.pl

```diff
@@ -261,10 +261,15 @@
       my ( $do_request_json_for, $await_event_for ) = @_;
 
       provide make_test_room => sub {
-         my ( $creator, @other_members ) = @_;
+         # First member is the room creator
+         my ( $members, %options ) = @_;
+
+         $options{visibility} = "public" unless exists $options{visibility};
```

Comment:
This is usually written using the definedor-equals operator;

    $options{visibility} //= "public";

technically that cares about definedness rather than actual existence in the hash, but it's usually best to treat a present-but-undef option the same as a missing one. It makes things easier to pass through other layers of function call.